package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "root",
-  "version": "3.53.1",
+  "version": "4.0.0",
```

```suggestion
  "version": "4.0.0-beta.0",
```

Will just need to make this a beta version for now, so that the change doesn't override if other people want to update components on the master branch.